### PR TITLE
[Fix-5250][API] Fixed data source parameter type attribute being null

### PR DESCRIPTION
**Pull request by Jonathan-Wei**
## Purpose of the pull request

Fixed data source parameter type attribute being null
#5250

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/DataSourceServiceImpl.java [hunk 1]
```diff
@@ -488,7 +488,6 @@ public String buildParameter(DbType type, String host,
             separator = ";";
         }
 
-        parameterMap.put(TYPE, connectType);
```
Comment (Jonathan-Wei) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/DataSourceServiceImpl.java:491]: Deletes type parameter assignment

**Review by Jonathan-Wei — COMMENTED** [review]
Deletes type parameter assignment

**Review by Jonathan-Wei — COMMENTED** [review]
Fixed test case issues related to deleting the type parameter

**Review by Jonathan-Wei — COMMENTED** [review]
Fixed data source parameter type attribute being null

**Comment by davidzollo** [comment]
@ruanwenjun please review this PR, thanks

**Review by ruanwenjun — APPROVED** [review]
Looks good to me :)

**Review by zhuangchong — APPROVED** [review]
+1

**Pull request merged**